geoengine/types.py

```diff
@@ -85,6 +85,9 @@ def from_response(response: Dict[str, Any]) -> None:
         if 'columns' in response:
             return VectorResultDescriptor(response)
 
+        if len(response) == 0:
```

Comment:
is a type available with the enum tag changes?

Reply:
79e85d5